### ISSUE TITLE
Remove SQLite specific rendering of quoted identifiers, to behave like other dialects

### ISSUE THREAD

**Issue by marceloverdijk** [jOOQ/jOOQ#16494]
### Use case

As discussed here https://groups.google.com/g/jooq-user/c/rCN6Idq28yc/m/2Nz18nWDBAAJ, currently the SQLite dialect does not seem to render quoted identifiers.

### Possible solution

Change back behavior not rendering quotes.

### jOOQ Version

3.19.6

### Database product and version

SQLite

### Java Version

17

**Comment by lukaseder** [comment]
Looking into this now. Having some fun with some integration tests, due to this behaviour:

```sql
create table "t" (
  "i" int
);

alter table "t" rename column "i" to "j";
insert into "t" values (1);
select "i", "j" from "t";
```

Resulting in:

```
|"i"|j  |
|---|---|
|i  |1  |
```

If a quoted identifier doesn't resolve, it's simply a string literal, in SQLite. 😅

**Comment by lukaseder** [comment]
I guess backticks are a better way to quote identifiers, then:

```sql
select `j` from `t`;
```

Results in:

```
|j  |
|---|
|1  |
```

Whereas

```sql
select `i` from `t`;
```

Results in the expected:

> SQL Error [1]: [SQLITE_ERROR] SQL error or missing database (no such column: i)

**Comment by lukaseder** [comment]
Issue changing the quoting from using double quotes to backticks in SQLite:

- https://github.com/jOOQ/jOOQ/issues/16598

**Comment by lukaseder** [comment]
Implemented in jOOQ 3.20.0
